test/framework/suite.py

```diff
@@ -96,7 +97,7 @@
 
 # call suite() for each module and then run them all
 # note: make sure the options unit tests run first, to avoid running some of them with a readily initialized config
-tests = [o, r, ef, ev, ebco, ep, e, mg, m, mt, f, run, a, robot, b, v, g, tcv, tc, t, c, s, l, f_c, sc, tw]
+tests = [o, r, ef, ev, ebco, ep, e, mg, m, mt, f, run, a, robot, b, v, g, tcv, tc, t, c, s, l, f_c, sc, tw, d, p]
```

Comment:
Jenkins isn't happy because of a broken test in `p`, so I suggest we flesh that out in a standalone PR (I'll tackle that bit, hopefully)